packages/dd-trace/src/platform/node/profiler.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const log = require('../../log')
+const { profiler, AgentExporter } = require('../../profiling')
+
+module.exports = function () {
```

Comment:
It allows the platform to do something before returning the profiler. While it's not used right now, it could be used later or by the browser platform.

Reply:
Actually, it's a function because it refers to `this` as being the platform, so it has to be a method on `platform`.

Reply:
I hadn't seen that. 

In that case, how about leaving it as-is, but doing a `const platform = this` and a comment indicating what a platform is (i.e. by referring to the file it's defined in).

Reply:
This code as evolved poorly, I don't think comments would help all that much and it should be refactored instead. I can try to improve at least this part of the platform if you prefer to make it cleaner.

Reply:
🤷 whatever makes the most sense.